gbus/worker.go

```diff
@@ -281,13 +306,18 @@ func (worker *worker) processMessage(delivery amqp.Delivery, isRPCreply bool) {
 		tx, txErr = worker.txProvider.New()
 		if txErr != nil {
 			worker.log().WithError(txErr).Error("failed to create transaction")
+			span.LogFields(slog.Error(txErr))
 			//reject the message but requeue it so it gets redelivered until we can create transactions
-			worker.Reject(true, delivery)
+			err = worker.Reject(true, delivery)
```

Comment:
It's probably worth invoking it with SafeWithRetries

Reply:
worker.Reject already implements safe retries https://github.com/wework/grabbit/blob/fix-opentracing/gbus/worker.go#L192